### ISSUE TITLE
DataTransfer Constructor not detected for browsers like IE 10

### ISSUE THREAD

**Issue by orosro** [amilajack/eslint-plugin-compat#509]
Hello,

DataTransfer constructor is not detected. The other rules work ok. My concern is that there might be other rules that are not detected, I saw that there are similar tickets. Is there any way of updating the list?

MDN: https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/DataTransfer

Example code
```
const dt = new DataTransfer();
```

Eslint config:
```
	"settings": {
		"react": {
			"version": "detect"
		},
		"lintAllEsApis": true,
		"polyfills": [
			"Promise",
			"fetch"
		]
	},
```
